.github/workflows/publish.yml

```
@@ -65,3 +65,11 @@ jobs:
             helm push "${pkg}" oci://ghcr.io/"${GITHUB_REPOSITORY_OWNER}"/charts |& tee .digest
             cosign sign $(cat .digest | awk -F "[, ]+" '/Pushed/{print $NF}')
           done
+      - uses: oras-project/setup-oras@v1
+        with:
+          oras-version: 1.2.0
```

```suggestion
          version: 1.2.0
```

I missed this in first go. This should be version as per the docs. 
Ref: https://github.com/oras-project/setup-oras?tab=readme-ov-file#advanced-usage

Night time for me as well :)

---

This time it was autocomplete on my side...
A clear sign to turn that off ;)
Thanks again!